View.php

```diff
@@ -176,6 +176,7 @@ protected function processMinifyCss($files)
             $css = '';
 
             foreach ($files as $file => $html) {
+                $file = (strpos($file, "?")) ? strstr($file, "?", true) : $file;
```

Comment:
In my opinion not the best solution. It is better to use the function `parse_url`
```php
$file = (strpos($file, "?")) ? parse_url($file, PHP_URL_PATH) : $file;
```